extensions/amp-ad-exit/0.1/config.js

```diff
@@ -16,6 +16,7 @@
 
 import {user} from '../../../src/log';
 import {FilterType} from './filters/filter';
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.